portfolio/src/main/webapp/script.js

```diff
@@ -15,34 +15,58 @@
 // limitations under the License.
  
 /**
- * Adds a random greeting to the page.
+ * Gives the user a random book recommendation.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
- 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
- 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+
  
-function sayRandomQuote(){
-    const quotes = [
-        "Elementary, my dear Watson",
-        "You miss 100% of the shots you don't take." ,
-        "The greatest glory in living lies not in never falling, but in rising every time we fall.",
-        "The future belongs to those who believe in the beauty of their dreams.",
-        "Do not go where the path may lead, go instead where there is no path and leave a trail." ,
-        "Life is really simple, but we insist on making it complicated." ,
+function sayRandomBook(){
+    const books = [
+        "Jane Eyre", "1984","Don Quixote", "The Great Gatsby", "Harry Potter and the Philosopher's Stone",
+        "War and Peace", "The Alchemist", "The Lord of the Rings", "Things fall apart", "Lord of the Flies", "Animal Farm",
+        "The Lion, The Witch, and the Wardrobe","Percy Jackson and the Lightning Thief", "Honor among thieves", "Summer Nights at the Midnight Hotel", "It", "The Hound of the Baskervilles",
     ];
  
-    const chosenQuote= quotes[Math.floor(Math.random() * quotes.length)];
+    const chosenbook= books[Math.floor(Math.random() * books.length)];
```

Comment:
I think javascript style is to use camelCase? Or at least the rest of the file is in camelCase, so use chosenBook here.